src/interfaces/ISablierMerkleVCA.sol

```diff
@@ -24,22 +24,25 @@ interface ISablierMerkleVCA is ISablierMerkleBase {
     /// value is denominated as a fixed-point number where 1e18 is 100%.
     function UNLOCK_PERCENTAGE() external view returns (UD60x18);
 
-    /// @notice Retrieves the timestamp when the airdrop token unlocks 100%.
+    /// @notice Retrieves the time when the VCA airdrop is fully vested, as a Unix timestamp.
     function VESTING_END_TIME() external view returns (uint40);
 
-    /// @notice Retrieves the timestamp when the airdrop token begins to unlock.
+    /// @notice Retrieves the time when the VCA airdrop begins to unlock, as a Unix timestamp. Zero is a sentinel value
```

Comment:
No, zero is not a sentinel value in MerkleVCA.

Reply:
Good catch!

Reply:
Feel free to push and merge

Reply:
https://github.com/sablier-labs/airdrops/pull/137/commits/627dfcaec09eff3ede42cf814b57e0b831ac77bc